File: Array/TargetSum.java
public class TargetSum {
    public static void main(String[] args){
        int [] arr = {3, 5, 1, 8, 2, 7};
        int target = 10;
        boolean found = false;
        for(int i = 0 ; i<arr.length; i++){
            for(int j = i+1; j<arr.length; j++){
                if(target==arr[i]+arr[j]){
                    System.out.println("sum found with elements "+ arr[i] +" "+ arr[j]);
                }
            }
        }
        

    }
}

File: Array/targetSum.py
arr = [1,2,3,4,5,6,7,8,9]
target = 10;
for i in range(length(arr)):
    for j in range(i+1, arr):
        if(target == arr[i] +arr[j]):
            print("targe sum found ", arr[i], "and" , arr[j])

